python/plugins/db_manager/dlg_sql_window.py

```diff
@@ -91,10 +91,8 @@ def __init__(self, iface, db, parent=None):
         self.updatePresetsCombobox()
 
         self.geomCombo.setEditable(True)
-        self.geomCombo.lineEdit().setReadOnly(True)
 
         self.uniqueCombo.setEditable(True)
-        self.uniqueCombo.lineEdit().setReadOnly(True)
```

Comment:
Under what circumstances would the user want to enter a value that is not a column in ```geomCombo```or ```uniqueCombo```? Or actually a list of columns for the latter, if the provider supports composite keys (currently only Postgres does).